### ISSUE TITLE
[Bug] A1111`s webui dev launch error

### ISSUE THREAD

**Issue by rltgjqmcpgjadyd** [Zyin055/Config-Presets#56]
```
[ERROR][Config Presets] The txt2img component 'txt2img_restore_faces' could not be processed. This may be because you are running an outdated version of the Config-Presets extension, or you included a component ID in the custom tracked components config file that does not exist, no longer exists (if you updated an extension), or is an invalid component (if this is the case, you need to manually edit the config file at J:\SDWebUI\extensions\Config-Presets\config-txt2img-custom-tracked-components.txt or just delete it so it resets to defaults). This extension will not work until this issue is resolved.

[ERROR][Config Presets] The img2img component 'img2img_restore_faces' could not be processed. This may be because you are running an outdated version of the Config-Presets extension, or you included a component ID in the custom tracked components config file that does not exist, no longer exists (if you updated an extension), or is an invalid component (if this is the case, you need to manually edit the config file at J:\SDWebUI\extensions\Config-Presets\config-img2img-custom-tracked-components.txt or just delete it so it resets to defaults). This extension will not work until this issue is resolved.

```

deleting the settings txt doesn't work

**Comment by Zyin055** [comment]
You need to give more context.

What version of Config Presets are you using? The update that was just released last night?

A1111 or SD.Next?

What do you think caused it to break?

**Comment by rltgjqmcpgjadyd** [comment]
use latest dev branch A1111`s webui & latest config preset

this seems to be a problem caused by a lot of UI changes in dev

**Comment by Zyin055** [comment]
I'll wait until their changes are pushed to an official release to look into this further, since stuff in their dev branch is likely to change.

**Comment by Zyin055** [comment]
Please see if the most recent update fixed the issue for 1.6.0.
